ods_ci/tests/Resources/Common.robot

```diff
@@ -46,6 +46,16 @@ End Web Test
     END
     Close Browser
 
+End Non JupyterLab Web Test
```

Comment:
maybe this could fit more at `ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot `

Reply:
I can move it, although I copied and amended method above named `End Web Test`, that's why I kept it here. Let me know if you still prefer me to move it.